shared/lib/snaps/snaps.ts

```diff
@@ -0,0 +1,21 @@
+import { SnapId } from '@metamask/snaps-sdk';
+
+export const PREINSTALLED_SNAPS = [
+  'npm:@metamask/message-signing-snap',
+  'npm:@metamask/ens-resolver-snap',
+  'npm:@metamask/institutional-wallet-snap',
+  'npm:@metamask/account-watcher',
+  'npm:@metamask/preinstalled-example-snap',
+  'npm:@metamask/bitcoin-wallet-snap',
+  'npm:@metamask/solana-wallet-snap',
+];
```

Comment:
is this not somewhere else on the codebase?

Reply:
it is something which was changed in main after the release was cut, there is a couple of stuff like that